DQMServices/Components/src/DQMScalInfo.cc

```diff
@@ -25,6 +31,7 @@ DQMScalInfo::DQMScalInfo(const edm::ParameterSet& ps)
   gtCollection_        = consumes<L1GlobalTriggerReadoutRecord>(parameters_.getUntrackedParameter<std::string>("gtCollection","gtDigis"));
   dcsStatusCollection_ = consumes<DcsStatusCollection>(parameters_.getUntrackedParameter<std::string>("dcsStatusCollection","scalersRawToDigi"));
   l1tscollectionToken_ = consumes<Level1TriggerScalersCollection>(parameters_.getUntrackedParameter<std::string>("l1TSCollection", "scalersRawToDigi"));
+  lumicollectionToken_ = consumes<LumiScalersCollection>(parameters_.getUntrackedParameter<std::string>("lumiCollection", "scalersRawToDigi"));
```

Comment:
this should be an InputTag not a string. (I guess that is true for all of them, but we should not make the problem bigger)